attestation-agent/attestation-agent/src/config/mod.rs

```diff
@@ -16,11 +19,66 @@ pub mod kbs;
 
 pub const DEFAULT_AA_CONFIG_PATH: &str = "/etc/attestation-agent.conf";
 
+pub const DEFAULT_EVENTLOG_HASH: &str = "sha384";
+
+pub const DEFAULT_EVENTLOG_PATH: &str = "/run/confidential-containers/eventlog";
+
+/// Hash algorithms used to calculate runtime/init data binding
+#[derive(Deserialize, Clone, Debug)]
+#[serde(rename_all = "lowercase")]
+pub enum HashAlgorithm {
+    Sha256,
+    Sha384,
+    Sha512,
+}
+
+impl Default for HashAlgorithm {
+    fn default() -> Self {
+        Self::Sha384
+    }
+}
+
+impl HashAlgorithm {
+    pub fn digest(&self, material: &[u8]) -> Vec<u8> {
+        match self {
+            HashAlgorithm::Sha256 => {
+                let mut hasher = Sha256::new();
+                hasher.update(material);
+                hasher.finalize().to_vec()
+            }
+            HashAlgorithm::Sha384 => {
+                let mut hasher = Sha384::new();
+                hasher.update(material);
+                hasher.finalize().to_vec()
+            }
+            HashAlgorithm::Sha512 => {
+                let mut hasher = Sha512::new();
+                hasher.update(material);
+                hasher.finalize().to_vec()
+            }
+        }
+    }
+}
+
 #[derive(Clone, Debug, Deserialize, Default)]
 pub struct Config {
     /// configs about token
     pub token_configs: TokenConfigs,
-    // TODO: Add more fields that accessing AS needs.
+
+    /// configs about eventlog
+    pub eventlog_config: EventlogConfig,
+}
+
+#[derive(Clone, Debug, Deserialize, Default)]
+pub struct EventlogConfig {
+    /// Hash algorithm used to extend runtime measurement for eventlog.
+    pub eventlog_algorithm: HashAlgorithm,
+
+    /// Path to store eventlog.
+    pub eventlog_path: String,
```

Comment:
we may need implement self defined `Default` for `EventlogConfig` and assign the default eventlog path, else the path may empty.